src/RecoveryServices/RecoveryServices/ChangeLog.md

```diff
@@ -19,6 +19,11 @@
 -->
 ## Upcoming Release
 
+## Version 2.2.0
+
+* Azure Site Recovery support for Azure Disk Encryption one pass for Azure to Azure..
```

Comment:
Please append your changelog to the `upcomming release` title. Don't add your version title.

Reply:
added to upcoming release